### ISSUE TITLE
Small Bug in PPO with GnTRedo

### ISSUE THREAD

**Issue by AIBoticsGmbH** [shibhansh/loss-of-plasticity#14]
In the value-side ReDo branch, the code checks elif isinstance(self.pol_gnt, GnTredo)
Instead of elif isinstance(self.val_gnt, GnTredo)
That looks like a bug. But I'm not 100% sure so please check
Context:
```
                # Selective reinitialization
                if self.pgnt:
                    with torch.no_grad():
                        if isinstance(self.pol_gnt, GnT):
                            self.pol_gnt.gen_and_test(features=self.pol.get_activations()+[None])
                        elif isinstance(self.pol_gnt, GnTredo):
                            self.pol_gnt.gen_and_test(features_history=torch.stack(self.pol.get_activations()).permute(1, 0, 2))
                if self.vgnt:
                    with torch.no_grad():
                        if isinstance(self.val_gnt, GnT): 
                            self.val_gnt.gen_and_test(features=self.vf.get_activations()+[None])
                        elif isinstance(self.pol_gnt, GnTredo): =>>>>> here it should be self.val_gnt instead of self.pol_gnt
                            features_history = torch.stack(self.vf.get_activations()).permute(1, 0, 2)
                            self.val_gnt.gen_and_test.....
```
https://github.com/shibhansh/loss-of-plasticity/blob/main/lop/algos/rl/ppo.py#L162

**Comment by shibhansh** [comment]
You're right. This is a small bug. However, it does not change any results as long as both value and policy networks use redo, which is the case in all the experiments we conducted.
Please, send a pull request with the bug fix :)
